cmd/erasure.go

```diff
@@ -434,6 +435,13 @@ func (er erasureObjects) nsScanner(ctx context.Context, buckets []BucketInfo, bf
 		}
 	}()
 
+	// Restrict parallelism for disk usage scanner
+	// upto GOMAXPROCS if GOMAXPROCS is < len(disks)
+	maxProcs := runtime.GOMAXPROCS(0)
```

Comment:
Can we do this after disk suffling in L447 ?

Reply:
We can.